server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitWithAuth.java

```diff
@@ -230,7 +195,7 @@ public void beginTask(String title, int totalWork) {
         }
     }
 
-    private static final class MirrorHostKeyRepository implements HostKeyRepository {
+    protected static class MirrorHostKeyRepository implements HostKeyRepository {
```

Comment:
Should we revert `final`? No class extends `MirrorHostKeyRepository` and `MirrorIdentityRepository`

Reply:
Yes